fractal_tasks_core/tasks/napari_workflows_wrapper.py

```diff
@@ -194,32 +188,19 @@ def napari_workflows_wrapper(
             "We currently only support a single input path"
         )
     in_path = Path(input_paths[0]).as_posix()
-    num_levels = metadata["num_levels"]
-    coarsening_xy = metadata["coarsening_xy"]
     label_dtype = np.uint32
 
-    # Load zattrs file and multiscales
-    zattrs_file = f"{in_path}/{component}/.zattrs"
-    with open(zattrs_file, "r") as jsonfile:
-        zattrs = json.load(jsonfile)
-    multiscales = zattrs["multiscales"]
-    if len(multiscales) > 1:
-        raise NotImplementedError(
-            f"Found {len(multiscales)} multiscales, "
-            "but only one is currently supported."
-        )
-    if "coordinateTransformations" in multiscales[0].keys():
-        raise NotImplementedError(
-            "global coordinateTransformations at the multiscales "
-            "level are not currently supported"
-        )
-
-    # Read ROI table
+    # Read mROI table
```

Comment:
Great that this logic here gets removed from the task! It really should be part of the loading library! :)